LiftLog.Ui/Shared/Presentation/HistoryCalendar.razor

```diff
@@ -4,8 +4,11 @@
 @{
     var firstDayOfMonth = new DateOnly(CurrentYear, CurrentMonth, 1);
     var dayOfFirstDayOfTheMonth = (int)firstDayOfMonth.DayOfWeek;
-    // If the first day of the month is Sunday (0) we show no days from the previous month as it is a full week
-    var numberOfDaysToShowFromPreviousMonth = dayOfFirstDayOfTheMonth;
+    var firstDayOfWeek = CultureInfo.DefaultThreadCurrentCulture != null ? CultureInfo.DefaultThreadCurrentCulture.DateTimeFormat.FirstDayOfWeek : DayOfWeek.Sunday;
```

Comment:
```suggestion
    var firstDayOfWeek = CultureInfo.CurrentCulture.DateTimeFormat.FirstDayOfWeek;
```

Nit: Just the correct culture info (my mistake in the issue), which is never null